jetstream/maxtext/single-host-inference/deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: maxengine-server
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: maxengine-server
+  template:
+    metadata:
+      labels:
+        app: maxengine-server
+    spec:
+      nodeSelector:
+        cloud.google.com/gke-tpu-topology: 2x4
+        cloud.google.com/gke-tpu-accelerator: tpu-v5-lite-podslice
+      containers:
+      - name: maxengine-server
+        image: us-docker.pkg.dev/cloud-tpu-images/inference/maxengine-server:v1.0.0
+        securityContext:
+          privileged: true
+        args:
+        - model_name=gemma-7b
```

Comment:
Where are we getting the model checkpoint from?

Reply:
If this yaml wouldn't work without a checkpoint path, then it would be good to include it in the args here with a placeholder for the path `<GCS_model_checkpoint_path>` to make it explicit. Also, are there instructions on where they can get the `gemma-7b` checkpoint for JetStream or how they can convert it from another format?

Reply:
I don't think we need to put the placeholder here, it will makes it the only thing to fail the deployment. ButI would agree we need to give a pointer about how to convert the checkpoint @vivianrwu

Reply:
The yaml works without a adding the checkpoint path.

Reply:
How? Are the model weights baked into the container image or are we downloading from somewhere?

Reply:
Yaml works without the checkpoint path but the responses with be nonsensical without a checkpoint. The model is baked into the container image from the maxtext repo - https://github.com/google/maxtext/blob/main/MaxText/configs/models/gemma-7b.yml

Reply:
again, the whole point for the user guide here is for guidance. We don't have a public checkpoint for use, so we should not include it in the yaml. This is clearly stated in the readme

Reply:
To clarify, I will add the directions to convert the checkpoint, which is running the script here - https://github.com/google/maxtext/blob/main/end_to_end/test_gemma.sh#L14. Right now, we are not providing a converted final checkpoint for users.

Reply:
Yep, thanks for the suggestion Ashok, we should update the checkpoint once we have one from CMCS later